File: src/ClubBundle/Entity/Membre.php
<?php

namespace ClubBundle\Entity;

use Doctrine\ORM\Mapping as ORM;
/**
 * Membre
 *
 * @ORM\Table(name="membre")
 * @ORM\Entity(repositoryClass="ClubBundle\Repository\MembreRepository")
 */
class Membre
{
    /**
     * @var int
     *
     * @ORM\Column(name="id", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $id;


    /**
     * Get id
     *
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }


    /**
     * @ORM\ManyToOne(targetEntity="club")
     *
     * @ORM\JoinColumn(name="idclub",referencedColumnName="idclub")
     */
    private $idclub;

    /**
     * @return mixed
     */
    public function getEleve()
    {
        return $this->eleve;
    }

    /**
     * @param mixed $eleve
     */
    public function setEleve($eleve)
    {
        $this->eleve = $eleve;
    }
    /**
     * @ORM\ManyToOne(targetEntity="eleve")
     *
     * @ORM\JoinColumn(name="ideleve",referencedColumnName="id")
     */
    private $eleve;

    /**
     * @ORM\ManyToOne(targetEntity="ClubBundle\Entity\User")
     *
     * @ORM\JoinColumn(name="id_user",referencedColumnName="id")
     */
    private $idUser;


    /**
     * Set idclub
     *
     * @param \ClubBundle\Entity\Club $idclub
     *
     * @return Membre
     */
    public function setIdclub(\ClubBundle\Entity\Club $idclub = null)
    {
        $this->idclub = $idclub;

        return $this;
    }

    /**
     * Get idclub
     *
     * @return \ClubBundle\Entity\Club
     */
    public function getIdclub()
    {
        return $this->idclub;
    }

    /**
     * Set idUser
     *
     * @param \ClubBundle\Entity\User $idUser
     *
     * @return Membre
     */
    public function setIdUser(\ClubBundle\Entity\User $idUser = null)
    {
        $this->idUser = $idUser;

        return $this;
    }

    /**
     * Get idUser
     *
     * @return \ClubBundle\Entity\User
     */
    public function getIdUser()
    {
        return $this->idUser;
    }



}

File: pidev/src/entities/Membre.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package entities;

/**
 *
 * @author User
 */
public class Membre {
    private int id;
    private int idclub;
    private int ideleve;
    private int id_user;
private String parent;
private String club;
private String eleve;

    public String getParent() {
        return parent;
    }

    public void setParent(String parent) {
        this.parent = parent;
    }

    public String getClub() {
        return club;
    }

    public void setClub(String club) {
        this.club = club;
    }

    public String getEleve() {
        return eleve;
    }

    public void setEleve(String eleve) {
        this.eleve = eleve;
    }

    public Membre(int id, int idclub, int ideleve, int id_user) {
        this.id = id;
        this.idclub = idclub;
        this.ideleve = ideleve;
        this.id_user = id_user;
    }

     public Membre() {}

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getIdclub() {
        return idclub;
    }

    public void setIdclub(int idclub) {
        this.idclub = idclub;
    }

    public int getIdeleve() {
        return ideleve;
    }

    public void setIdeleve(int ideleve) {
        this.ideleve = ideleve;
    }

    public int getId_user() {
        return id_user;
    }

    public void setId_user(int id_user) {
        this.id_user = id_user;
    }
}
